tools/functional-tester/etcd-tester/cluster.go

```diff
@@ -217,6 +164,22 @@ func (c *cluster) Status() ClusterStatus {
 	return cs
 }
 
+func (c *cluster) maxRev() (rev int64, err error) {
+	for _, m := range c.Members {
+		curRev, _, curErr := m.RevHash()
+		if curErr != nil {
+			err = curErr
+		}
+		if curRev > rev {
+			rev = curRev
+		}
+	}
+	if rev == 0 {
```

Comment:
If this func is meant to fix max rev, we should return error when there is any. Current logic might mask errors.